app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+
+  def flash_class(level)
+    case level
+        when :notice then "alert alert-info"
```

Comment:
I wish I could take credit for the code, I found it online when I was researching how to format the messages for Bootstrap.

I like to stick with convention, I will make the change.